src/main/resources/openshift/builder/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM jboss-eap-7/eap70-openshift:1.4
+FROM jboss-eap-7/eap70-openshift:latest
```

Comment:
I'm not sure if it is really good idea to use `latest` tag. It is considered a bad practice (https://developers.redhat.com/blog/2016/02/24/10-things-to-avoid-in-docker-containers/) 

With latest we cannot have any control of used version, so when some incompatible version is released, it would also break all of our code.